react/features/recording/components/Recording/native/RecordingConsentDialog.tsx

```diff
@@ -31,8 +32,8 @@ export default function RecordingConsentDialog() {
     const consentAndUnmute = useCallback(() => {
         dispatch(setAudioUnmutePermissions(false, true));
         dispatch(setVideoUnmutePermissions(false, true));
-        dispatch(setAudioMuted(false));
-        dispatch(setVideoMuted(false));
+        dispatch(setAudioMuted(false, true));
+        dispatch(setVideoMuted(false, VIDEO_MUTISM_AUTHORITY.USER, true));
```

Comment:
How about screenshare? Are we ok with sharing screen without consent when recording starts?